src/styles/_catalog.scss

```
@@ -1,22 +1,44 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
 
   max-width: 944px;
   margin: 0 auto;
+  gap: 48px 48px;
```

```suggestion
  gap: 48px;
```

The same but shorter